net/strongswan/files/swanctl.init

```diff
@@ -684,6 +707,17 @@ prepare_env() {
 	config_load ipsec
 	config_foreach config_ipsec ipsec
 	config_foreach config_remote remote
+	
+	swanctl_xappend0 "# Global config"
+	swanctl_xappend0 ""
+	
+	swanctl_xappend0 "pools {"
+	config_foreach config_pool pools
```

Comment:
If I'm reading this correctly, we're changing the section that `pools` are defined in.  Won't that break existing configurations without a migration script being applied?

Reply:
See the text of commit 23ccb705184de16aef8001bc183d0786e1e23cde . I don't remember exactly what I did 1 year ago, but if the commit message is right, this change is not breaking anything.